src/reducers/collections.js

```diff
@@ -35,7 +35,7 @@ const selectors = {
       return collection.get('fields');
     },
     entryPath(collection, slug) {
-      return `${ collection.get('folder').replace(/\/$/, '') }/${ slug }.${ this.entryExtension(collection) }`;
+      return `${ collection.get('folder').replace(/\/$/, '') }/${ slug.indexOf(collection.get('name')) == 0 ? slug.split("-").slice(1).join("-") : slug }.${ this.entryExtension(collection) }`;
```

Comment:
this code checks if the collection name is in the file and removes it(since the files are always saved without the collection name), because the file is usually based on the slug name, but file is created before i modify the slug, this makes the code compatible with both versions of `collection-slug` and simply `slug`

Reply:
Not editing the slug allows us to avoid this change.